mappings/net/minecraft/client/render/BackgroundRenderer.mapping

```diff
@@ -1,18 +1,59 @@
 CLASS net/minecraft/unmapped/C_pdccygbj net/minecraft/client/render/BackgroundRenderer
 	FIELD f_dfpsklbj blue F
+	FIELD f_gkvccaki FOG_TRANSITION_TIME_MS F
```

Comment:
The usage of MS vs Milliseconds is inconsistent, could one or the other be picked?